Sources/scipio/Arguments.swift

```diff
@@ -10,19 +10,6 @@ extension URL: @retroactive ExpressibleByArgument {
     }
 }
 
-extension BuildConfiguration: @retroactive ExpressibleByArgument {
-    public init?(argument: String) {
-        switch argument.lowercased() {
-        case "debug":
-            self = .debug
-        case "release":
-            self = .release
-        default:
-            return nil
-        }
-    }
-}
-
```

Comment:
This extension was also required for Swift 5 & 6.

Please move L34 to the outside of the preprocessor block.

Reply:
https://github.com/giginet/Scipio/blob/c062eeb00bfd62618a7e316cab725a318262c77b/Sources/scipio/Arguments.swift

The `BuildConfiguration` extension is already outside the preprocessor block.
If my perception is wrong, I would like to know!

Reply:
oops. It's correct!